explorer/Explorer.scss

```diff
@@ -30,10 +30,10 @@ html.IsInIframe #ExplorerContainer {
     justify-content: space-around;
 
     .ExplorerTitle {
-        font-size: 28px;
+        font-size: clamp(16px, 1.5vw, 19px);
+        line-height: clamp(18px, 1.7vw, 23px);
```

Comment:
One potential thing to check here – for browsers that [don't support `clamp`](https://caniuse.com/css-math-functions), should we have a fallback? Or are those browsers not supported anyway (due to JavaScript requirements)?

I think fallbacks are defined by setting multiple properties, e.g. 

```css
font-size: 19px;
font-size: clamp(16px, 1.5vw, 19px);
```

Then if the bottom one is not supported, the top one is used?

Reply:
Ahh... do we have an official list of browser ranges we support? 😄

Reply:
Not really - I guess the unofficial rule is "as many as we can _easily_ support".

However, I checked what this looks like in Firefox 53 (which doesn't have support for `clamp`) and it looks alright. In my opinion we don't absolutely need to have fallback styles _in this case_.

But good thinking and something to keep in mind for the next time around!

![image](https://user-images.githubusercontent.com/2641501/119673756-e76d9b00-be3b-11eb-8f67-f0599531edb4.png)